bool take_single_measurement
---
bool success